### DISCUSSION TITLE
Welcome to xpoc-framework Discussions!

### DISCUSSION THREAD

**Discussion by christianpaquin** [microsoft/xpoc-framework#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Forgetmenot1** [comment]
changes.md I will be there for the whole vibe I will be fine and you acting as I dive in the first place Wouldn't you want that person to know how it began to be with myself and think about it for a while tearing piece by peice away not to fast though Accordingly doing it for a while tearing piece by peice overtime to get ahold of this platform  I need help
